common/src/main/java/org/mcaccess/minecraftaccess/mixin/NarrationThunkMixin.java

```diff
@@ -0,0 +1,32 @@
+package org.mcaccess.minecraftaccess.mixin;
+
+import net.minecraft.client.gui.narration.NarrationThunk;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Mutable;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+
+@Mixin(NarrationThunk.class)
+public class NarrationThunkMixin<T> {
+    @Mutable
+    @Final
+    @Shadow
+    private T contents;
+
+    /**
+     * <a href="https://minecraft.wiki/w/Formatting_codes">wiki page of formatting codes</a>
+     */
+    @SuppressWarnings("unchecked")
+    @Inject(at = @At("TAIL"), method = "<init>")
+    private void removeFormattingCodes(Object value, BiConsumer<Consumer<String>, T> transformer, CallbackInfo ci) {
```

Comment:
Shouldn't the type of `value` be `T`?
```suggestion
    private void removeFormattingCodes(T value, BiConsumer<Consumer<String>, T> transformer, CallbackInfo ci) {
```